mac/Info-make.plist

```diff
@@ -12,15 +12,24 @@
         <string>APPL</string>
         <key>CFBundleVersion</key>
         <string>@FULL_VERSION@</string>
+        <key>CFBundleShortVersionString</key>
+	    <string>@SHORT_VERSION@</string>
+        <key>CFBundleSupportedPlatforms</key>
+	    <array>
+		    <string>MacOSX</string>
+        </array>
+        <key>LSApplicationCategoryType</key>
+	    <string>public.app-category.music</string>
+        <key>LSMinimumSystemVersion</key>
+        <string>10.13.0</string>
```

Comment:
This is only for the Xcode build. However, we should ensure to still provide support for old versions of macOS (legacy build). However, there's no big need to sign it.

Reply:
This isn’t about signing. These are required parameters for the App Store processing.

Reply:
Yes. So the version provided by the app store would only allow us to support macOS 10.13 or higher. Currently the "legacy" version supports macOS Sierra (10.12) or lower. I don't think it's a big deal here.

Reply:
Yes, that is correct. I’d be interested to know how many people actually run anything that old…

Reply:
I do know some people ;-). 

One of my choir leaders has been on mavericks until the beginning of this year.

Reply:
Amazing